cmd/drone-docker-ecr/main.go

```diff
@@ -75,6 +75,10 @@ func main() {
 	}
 }
 
+func getRepoName(repo string) string {
+	return strings.SplitN(repo, "/", 2)[1]
```

Comment:
this could cause an index out-of-bounds panic

Reply:
Thanks for catching this.